File: Daily/1291. Sequential Digits/Solution.py
from typing import List
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        sequential_Numbers, digits = set(), '123456879'
        for length in range(len(str(low)), len(str(high)) + 1):
            for index in range(10 - length + 1):
                num = int(digits[index:index + length])
                if low <= num <= high: sequential_Numbers.add(num)
        return sorted(list(sequential_Numbers))
if __name__ == '__main__':
    low, high = (map(int, input().split())); sol = Solution()
    print('Sequential Numbers between', low, 'and', high, ':', sol.sequentialDigits(low, high))

File: Daily/1291. Sequential Digits/Solution.cpp
#include <iostream>
#include <vector>
#include <functional>
#include <string>
#include <set>
using namespace std;
class Solution {
    public:
        vector<int> sequentialDigits(int low, int high) {
            set<int> sequential_Numbers; string digits = "123456789";
            for(int length = to_string(low).size(); length <= to_string(high).size(); ++length) {
                for(int index = 0; index <= 10 - length; ++index) {
                    string numStr = digits.substr(index, length);
                    int num = stoi(numStr);
                    if(num >= low && num <= high) sequential_Numbers.insert(num);
                }
            }
            return vector<int>(sequential_Numbers.begin(), sequential_Numbers.end());
        }
};
int main() {
    function<void(vector<int>)> printVector = [&](vector<int> array) -> void {
        cout << '[';
        for(int index = 0; index < array.size(); ++index) {
            cout << array[index];
            if(index < array.size()-1) cout << ',';
        }
        cout << ']' << endl;
    };
    int low, high; Solution sol; cin >> low >> high;
    vector<int> sequential_Numbers = sol.sequentialDigits(low, high);
    cout << "All Sequential Numbers between " << low << " and " << high << ": ";
    printVector(sequential_Numbers);
}